contracts/flashMint/flash.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Why do we even need that file ?
shouldn't we just use 
IERC3156FlashLender interface instead ?